setup.py

```diff
@@ -1,14 +1,21 @@
 #!/usr/bin/env python
 from __future__ import absolute_import
 
+import logging
 import os
 import sys
 
-if os.environ.get("SENTRY_PYTHON2") == "1" and sys.version_info[:2] != (2, 7):
+logger = logging.getLogger()
+
+version = sys.version_info
+if os.environ.get("SENTRY_PYTHON2") == "1" and version[:2] != (2, 7):
     sys.exit("Error: Sentry [In DEPRECATED Python 2 mode] requires Python 2.7.")
 
-if os.environ.get("SENTRY_PYTHON2") != "1" and sys.version_info[:2] != (3, 6):
-    sys.exit("Error: Sentry requires Python 3.6.")
+if os.environ.get("SENTRY_PYTHON2") != "1" and version[0] == 3:
```

Comment:
Let's avoid doing things like `version[0] == 3` since this wouldn't be good if Python 4 came out.

You can perform arithmetic comparisons on the entire sys.version_info tuple.

Reply:
Thanks for catching this!